content/en/entities/Translation.md

```diff
@@ -99,8 +97,24 @@ Translation of status with poll:
 **Version history:**\
 4.0.0 - added
 
+## Translation::Poll attributes {#Poll}
+
+### `id` {#Poll-id}
+
+**Description:** The ID of the Poll.\
+**Type:** String (cast from an integer, but not guaranteed to be a number)\
+**Version history:**\
+4.2.0 - added
+
+### `options` {#Poll-options}
+
+**Description:** The translated poll options.\
+**Type:** Array\
```

Comment:
https://github.com/mastodon/mastodon/blob/main/app/serializers/rest/translation_serializer.rb

particularly https://github.com/mastodon/mastodon/blob/90eb4a5d01f1f1e8a76e7dc814eae992a9c1ee16/app/serializers/rest/translation_serializer.rb#L18-L20

i think this would be an `Array of Translation::Poll::Option`? and `Translation::Poll::Option` would have one attribute which is `title`?